games_report.txt

```diff
@@ -0,0 +1,177 @@
+Game 2:
```

Comment:
You got the report info, but the format is different than the expected (as json)

Reply:
so, to fix it i need to write this file in json synthax, yes?

Reply:
https://gist.github.com/talyssonoc/a51b5a6c42c8c60c0f8c0f28b34f8dbc

> Para cada jogo o parser deve gerar algo como:
```
game_1: {
    total_kills: 45;
    players: ["Dono da bola", "Isgalamido", "Zeh"]
    kills: {
      "Dono da bola": 5,
      "Isgalamido": 18,
      "Zeh": 20
    }
  }
  ```
  
  You should strictly follow the requirements.

Reply:
I mean, you did a great job, and your report looks good. But in our case, the format is part of the requirements.